src/Compilers/Core/Portable/Operations/Statement.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    internal partial class VariableDeclaration : IVariableDeclaration
+    {
+        public VariableDeclaration(ILocalSymbol variable, IOperation initialValue, SyntaxNode syntax) :
+            this(variable,
+                initialValue,
+                variable == null || (initialValue != null && initialValue.IsInvalid),
```

Comment:
Sure, it's just a nitpick anyway :)

---
In reply to: [109230042](https://github.com/dotnet/roslyn/pull/18318#discussion_r109230042) [](ancestors = 109230042)